testing-plugin/src/test/kotlin/arrow/meta/plugin/testing/ExampleTest.kt

```diff
@@ -14,8 +14,8 @@ class ExampleTest {
         | 
         """.source
       },
-      assert = {
```

Comment:
are we still preserving this? there are cases where we may want a single assert

Reply:
Good idea. thanks!! I'll change to have both options :+1:

Reply:
For instance, `assert` for a single assert and `assertAll` for multiple asserts.

Reply:
I recovered `assert` instead of `asserts` and I'll add the commented improvement into another PR :+1: